Directory.Build.props

```diff
@@ -28,6 +28,8 @@
   <Import Project="$(RepoRoot)/Directory.Build.props.user" Condition = "Exists('$(RepoRoot)/Directory.Build.props.user')" />
 
   <PropertyGroup Condition="'$(BUILDING_USING_DOTNET)' == 'true'">
+    <BUILDING_WITH_LKG>true</BUILDING_WITH_LKG>
```

Comment:
```suggestion
    <!-- LKG = Last Known Good -->
    <BUILDING_WITH_LKG>true</BUILDING_WITH_LKG>
```

I know that some people know.

Reply:
It's internal flag, do we really want to have a one-off comment? 
If we do, it needs to describe in details what does it mean for each release of dotnet.

Reply:
No I just noticed we have it a few more places in the codebase including the F# code itself so people might get confused. Could be somewhere in the readme though.

Reply:
Yeah, probably mentioning it in devguide(if not already) is better. Should be done separately though.

Reply:
Sure, I will be refreshing a few readmes soon so will put it there.